Sergey/Medium/style.css

```diff
@@ -0,0 +1,430 @@
+
+*
+{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body
+{
+    text-rendering: optimizeLegibility;
+    height: 2000px;
+    min-height: 100vh;
```

Comment:
Choose one of two: height or min-height.

Reply:
> Choose one of two: height or min-height.
It's must be width =\